dnase/analysis.sh

```diff
@@ -439,7 +439,6 @@ elif [[ "${sampleType}" == "dnase" ]] || [[ "${sampleType}" == "atac" ]] || [[ "
         fi
         #Kent tools can't use STDIN
         bedGraphToBigWig $TMPDIR/${name}.${mappedgenome}.perBase.clipped.bedGraph ${chromsizes} ${sampleOutdir}/${name}.${mappedgenome}.perBase.bw
-        
```

Comment:
stray line

Reply:
Not sure why line 442 looks that way on the right, in green.  There are only 2 tabs in the code.

Reply:
it looks like you added whitespace -- you're saying that's not the case?